Bio/PDB/Polypeptide.py

```diff
@@ -197,8 +197,11 @@ def get_ca_list(self):
         """
         ca_list = []
         for res in self:
-            ca = res["CA"]
-            ca_list.append(ca)
+            try:
```

Comment:
You can also just use the `dict.get` syntax to to this without the try/except. I would go for the try/except if you were to issue a warning.